docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -11,15 +11,11 @@ When scanning with Semgrep in CI, findings automatically populate in Semgrep Clo
 
 If you run the alternate job and it fails with a "resource not accessible by integration" error, there are two possible causes.
 
-## Your repository is private
+## Your repository's default workflow permissions are set to read-only
```

Comment:
A tad confused about this change - I think I'd stick with "repository's workflow permissions", mainly because they could be read-only by being the default (very likely) or for some other reason (were set that way intentionally after having been different), and I don't think that distinction is relevant here, unless I'm missing something (possible).